### PR TITLE
TRUNK-6214: Update OpenMRS version in fix_liquibase_snapshots.sh scri…

### PR DESCRIPTION
**Pull request by Seremba**
…pt to current release

## Issue I worked on

see https://openmrs.atlassian.net/browse/TRUNK-6214

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: liquibase/scripts/fix_liquibase_snapshots.sh [hunk 1]
```diff
@@ -17,6 +17,6 @@
 #  The updated snapshots are written to the openmrs-core/liquibase/snapshots folder.
 #
 
-openmrs_version=2.4.0-SNAPSHOT
+openmrs_version=2.6.0-SNAPSHOT
```
Comment (ManojLL) [inline liquibase/scripts/fix_liquibase_snapshots.sh:20]: we can extract the current OpenMRS version directly from the `pom.xml` file, So we no need to manually update the OpenMRS version during the release of a new version.

File: liquibase/scripts/fix_liquibase_snapshots.sh [hunk 2]
```diff
@@ -17,6 +17,9 @@
 #  The updated snapshots are written to the openmrs-core/liquibase/snapshots folder.
 #
 
-openmrs_version=2.4.0-SNAPSHOT
+pom_path="/mnt/c/Users/Admin/Desktop/development/openmrs-dev/openmrs-core/liquibase/pom.xml"
```
Comment (ManojLL) [inline liquibase/scripts/fix_liquibase_snapshots.sh:20]: I'd like to suggest using relative paths instead absolute path. Absolute paths can lead to issues when the project is moved to a different directory or when others are working on the project from different environments.

**Comment by Seremba** [comment]
Hi @ManojLL I am done adding the relative path

**Comment by Seremba** [comment]
Hi @ibacher @ManojLL @dkayiwa I finished this PR over three weeks ago, but I request its review. Would you like me to repeat it? Is there anything to correct? Thanks a lot.

**Comment by dkayiwa** [comment]
Can you include a link to the ticket as advised here? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by Seremba** [comment]
Thanks @dkayiwa , I am done.

**Comment by dkayiwa** [comment]
@Seremba did you test this and confirm that `openmrs_version` is assigned the appropriate value?

**Comment by Seremba** [comment]
Yes @dkayiwa, I surely did!

**Comment by dkayiwa** [comment]
How did you test it?

**Comment by Seremba** [comment]
> I `cd` into `openmrs\liquibase\srcipts` and then ran `bash fix_liquibase_snapshots.sh` then it printed this as shown in the picture. Screenshot 2024-02-22 183934 interest was in the fact that it displays the `openmrs_version`. That is how I tasted it @dkayiwa

The screenshot below is for the code. 
Screenshot 2024-02-22 184100

File: liquibase/scripts/fix_liquibase_snapshots.sh [hunk 3]
```diff
@@ -15,8 +15,8 @@
 #  Run this script from the openmrs-core/liquibase folder.
 #
 #  The updated snapshots are written to the openmrs-core/liquibase/snapshots folder.
-#
 
-openmrs_version=2.4.0-SNAPSHOT
 
-java -jar ./target/openmrs-liquibase-${openmrs_version}-jar-with-dependencies.jar
+openmrs_version=$(grep -m 1 '<version>' ../pom.xml | sed -n 's/.*<version>\(.*\)<\/version>.*/\1/p' | awk '{$1=$1;print}')
```
Comment (ibacher) [inline liquibase/scripts/fix_liquibase_snapshots.sh:20]:
```suggestion
openmrs_version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
```

**Comment by Seremba** [comment]
Hi @ibacher @dkayiwa @ManojLL

**Comment by Seremba** [comment]
Hello @dkayiwa @ManojLL and @ibacher, kindly review this. 
Screenshot 2024-03-01 090437
Screenshot 2024-03-01 090459

the two screenshots show that code extracts the `openmrs_version` from pom.xml using maven as I suggested by @ibacher. Thanks a lot!

**Comment by Seremba** [comment]
@dkayiwa @ibacher @ManojLL is there anything else I am meant to do on this PR?

File: liquibase/scripts/fix_liquibase_snapshots.sh [hunk 4]
```diff
@@ -15,8 +15,9 @@
...
 #  The updated snapshots are written to the openmrs-core/liquibase/snapshots folder.
-#
 
-openmrs_version=2.4.0-SNAPSHOT
 
-java -jar ./target/openmrs-liquibase-${openmrs_version}-jar-with-dependencies.jar
+openmrs_version=$(cd ../ && mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+
+
+java -jar ./target/openmrs-liquibase-"${openmrs_version}"-jar-with-dependencies.jar
```
Comment (dkayiwa) [inline liquibase/scripts/fix_liquibase_snapshots.sh:23]: What was the motivation behind putting `${openmrs_version}` in double quotes?

**Comment by Seremba** [comment]
I have removed the double quotes @dkayiwa

**Pull request merged**